test/react/login_form.test.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-undef */
+import * as React from "react";
+import LoginForm from "../../app/javascript/components/login_form";
+import { expect, test } from "@jest/globals";
+import { fireEvent, getByRole, queryByTestId, render, RenderResult } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import "@testing-library/jest-dom";
+import mockAxios from "jest-mock-axios";
+
+const renderForm = (
+  path = "/log",
```

Comment:
Setting this to be login would be a lot clearer

Reply:
Changed